src/playdate/build/pdxinfo.nim

```diff
@@ -0,0 +1,55 @@
+import std/[os, parsecfg, streams, strutils, strformat, times, osproc], nimbledump
+
+type PdxInfo* = object
+    ## Details used to populate the pdxinfo file
+    name*, author*, description*, bundleId*, imagePath*, version*, buildNumber*: string
+
+proc `$`*(pdx: PdxInfo): string =
+    for key, value in pdx.fieldPairs:
+        if value != "":
+            result &= key & "=" & value & "\n"
+
+proc write*(pdx: PdxInfo) =
+    ## Writes the pdxinfo file
+    createDir("source")
+    writeFile("source" / "pdxinfo", $pdx)
+
+proc join*(a, b: PdxInfo): PdxInfo =
+    ## Combins two PdxInfo instances
```

Comment:
This, too could be documented in the README: "to override specific fields, place a file called pdxinfo with your overrides in the root of your game's project"